api/kessel/relations/v1beta1/relation_tuples.proto

```diff
@@ -34,12 +34,26 @@ service KesselTupleService {
 			delete: "/v1beta1/tuples"
 		};
 	};
+	rpc ImportBulkTuples (stream
+			ImportBulkTuplesRequest) returns (ImportBulkTuplesResponse) {
+		option (google.api.http) = {
+			post: "/v1beta1/tuples/bulkimport"
+			body: "*"
+		};
+	};
+}
+
+message ImportBulkTuplesRequest {
+	repeated Relationship tuples = 1 [(buf.validate.field).required = true];
+}
+
+message ImportBulkTuplesResponse {
+	uint64 num_loaded = 1;
```

Comment:
```suggestion
	uint64 num_imported = 1;
```

Just for internal consistency (I know spicedb used "loaded" but it's consistent with the verb in the api–"import").